egs/aspire/s5/local/chain/copy_lat_dir.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Copyright 2016  Johns Hopkins University (author: Vijayaditya Peddinti)
+# Apache 2.0
+
+# This script operates on a directory, such as in exp/tri4a_lats,
+# that contains some subset of the following files:
+#  lat.*.gz
+#  tree
+#  cmvn_opts
+#  splice_opts
+#  num_jobs
+#  final.mdl
+# It copies to another directory, possibly adding a specified prefix or a suffix
+# to the utterance names.
+
+
+# begin configuration section
+cmd=run.pl
+utt_prefix=
+utt_suffix=
+# end configuration section
+
+. utils/parse_options.sh
+
+if [ $# != 2 ]; then
+  echo "Usage: "
+  echo "  $0 [options] <src_dir> <dest_dir>"
+  echo "e.g.:"
+  echo " $0  --utt-prefix=1- exp/tri4a_lat exp/tri4a_rev1_lat"
+  echo "Options"
+  echo "   --utt-prefix=<prefix>     # Prefix for utterance ids, default empty"
+  echo "   --utt-suffix=<suffix>     # Suffix for utterance ids, default empty"
+  exit 1;
+fi
+
+
+export LC_ALL=C
+
+src_dir=$1
+dest_dir=$2
+
+mkdir -p $dest_dir
+
+if [ ! -f $src_dir/lat.1.gz ]; then
+  echo "copy_lat_dir.sh: no such files $src_dir/lat.*.gz"
+  exit 1;
+fi
+
+for f in tree cmvn_opts splice_opts num_jobs final.mdl; do
+  if [ ! -f $src_dir/$f ]; then
+    echo "copy_lat_dir.sh: no such file $src_dir/$f this might be serious error."
+    continue
+  fi
+  cp $src_dir/$f $dest_dir/
+done
+
+nj=$(cat $dest_dir/num_jobs)
+mkdir -p $dest_dir/log
+
+$cmd JOB=1:$nj $dest_dir/log/copy_lat.JOB.log \
+  lattice-copy "ark:gunzip -c $src_dir/lat.JOB.gz |" ark,t:- \|\
+    awk -v prefix="$utt_prefix" -v suffix="$utt_suffix" '{if ((NF == 1) && (prev_nf == 0)) { printf("%s%s%s\n", prefix, $0, suffix) } else { print $0}; prev_nf=NF}' \|\
```

Comment:
@danpovey When copying a lattice directory which files should be definitely be copied other than `lat.*.gz tree cmvn_opts splice_opts num_jobs final.mdl` ?